### ISSUE TITLE
MASTERTSDatasetH fillna_type="ffill+bfill"是必要的吗？

### ISSUE THREAD

**Issue by LeonTing1010** [SJTU-Quant/MASTER#11]
在qlib的实现中MASTERTSDatasetH返回的TSDataSampler做了fillna_type="ffill+bfill"前后向填充，然而pytorch_master_ts.py中   
`    def fit(self, dataset: DatasetH, save_path=None):
        dl_train = dataset.prepare("train", col_set=["feature", "label"], data_key=DataHandlerLP.DK_L)
        dl_valid = dataset.prepare("valid", col_set=["feature", "label"], data_key=DataHandlerLP.DK_L)`没有配置`        dl_train.config(fillna_type="ffill+bfill")  # process nan brought by dataloader
        dl_valid.config(fillna_type="ffill+bfill")  # process nan brought by dataloader`，请问TSDataSampler中的fillna_type="ffill+bfill"是否是必要的

**Comment by keyboard1st** [comment]
您好，请问pytorch_master_ts.py文件在哪可以找到。十分感谢您的帮助，祝您生活愉快。

**Comment by LITONG99** [comment]
Please check the [SJTU-Quant qlib repo](https://github.com/SJTU-Quant/qlib) as we mentioned in the Readme.md. The file is  at qlib/contrib/model/pytorch_master_ts.py
